gcc/rust/ast/rust-ast-full-test.cc

```diff
@@ -4050,6 +4052,88 @@ Module::add_crate_name (std::vector<std::string> &names) const
     item->add_crate_name (names);
 }
 
+static bool
+check_for_file (std::string dir_name, std::string file_name)
```

Comment:
I think you want a static bool file_exists(const std::string path) 

this function is a bit more reuseable and it will be able to just look at a path and if it exists then check to make sure it is infact a file. It avoids having to do readdir and check if paths are in fact directories etc.

Reply:
Done in e0db5ce